### PR TITLE
convert temp coeff to %/C

### LINKED ISSUE #350: change units of temperature coefficient from 1/C to %/C
The temperature coefficient is currently specified in units of 1/C, so users input values like -0.003. This would be fine on its own, but we also have DC and AC loss factors that are specified in units of %. 

This was brought up by someone at the DOE review meeting in October but we forgot to create an issue for it. If I recall correctly, @cwhanse agreed that this should be changed. 

Probably now or never on this.

### PR DESCRIPTION
**Pull request by alorenzo175**
- [x] Closes #350 .
  - [x] I am familiar with the [contributing guidelines](https://solarforecastarbiter-core.readthedocs.io/en/latest/contributing.html).
  - [ ] Tests added.
  - [ ] Updates entries to [`docs/source/api.rst`](https://github.com/SolarArbiter/solarforecastarbiter-core/blob/master/docs/source/api.rst) for API changes.
  - [x] Adds descriptions to appropriate "what's new" file in [`docs/source/whatsnew`](https://github.com/SolarArbiter/solarforecastarbiter-core/tree/master/docs/source/whatsnew) for all changes. Includes link to the GitHub Issue with `` :issue:`num` `` or this Pull Request with `` :pull:`num` ``. Includes contributor name and/or GitHub username (link with `` :ghuser:`user` ``).
  - [x] New code is fully documented. Includes [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) compliant docstrings, examples, and comments where necessary.
  - [x] Maintainer: Appropriate GitHub Labels and Milestone are assigned to the Pull Request and linked Issue.

### CODE REVIEW

**Comment by alorenzo175** [comment]
API tests will probably fail until SolarArbiter/solarforecastarbiter-api#238

File: solarforecastarbiter/pvmodel.py [hunk 1]
```diff
@@ -277,6 +277,7 @@ def calculate_power(dc_capacity, temperature_coefficient, dc_loss_factor,
     ----------
     dc_capacity : float
     temperature_coefficient : float
+       Specified in units of %/C to be converted to 1/C
```
Comment (wholmgren) [inline solarforecastarbiter/pvmodel.py:280]:
```suggestion
        Specified in units of %/C to be converted to 1/C
```

**Comment by wholmgren** [comment]
oh, is there something that needs to be done about the reference data set?

**Comment by alorenzo175** [comment]
Yeah, adjusted here and the API migration (https://github.com/SolarArbiter/solarforecastarbiter-api/pull/238) will adjust those in the database already. Once this is in dev, I tag to prod relatively quickly so that any new sites a user makes are properly specified as %/C

**Comment by wholmgren** [comment]
> Once this is in dev, I want to tag to prod relatively quickly so that any new sites a user makes are properly specified as %/C

Does that mean a beta 6 as soon as possible?

**Pull request merged**